Interfaces/DBView.xib

```diff
@@ -1295,15 +1292,6 @@
                                                             <action selector="toggleCollapse:" target="7206" id="8024"/>
                                                         </connections>
                                                     </button>
-                                                    <button toolTip="Toggle between editing simple text cells as a spreadsheet or in pop-up sheets" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="8012">
-                                                        <rect key="frame" x="80" y="10" width="25" height="25"/>
-                                                        <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
-                                                        <buttonCell key="cell" type="smallSquare" bezelStyle="smallSquare" image="button_edit_modeTemplate" imagePosition="only" alignment="center" alternateImage="button_edit_mode_selectedTemplate" inset="2" id="8013">
-                                                            <behavior key="behavior" pushIn="YES" changeContents="YES" lightByContents="YES"/>
-                                                            <font key="font" metaFont="system"/>
-                                                        </buttonCell>
-                                                        <color key="contentTintColor" name="labelColor" catalog="System" colorSpace="catalog"/>
-                                                    </button>
```

Comment:
Is this removing the option to select the preferred editing mode? Why?

Reply:
It is because we have it in the modal + in main menu. The icon (unfortunately) didn't work and also caused the issues menitoned in #671